### PR TITLE
sqlmap: new, 1.7.12

### PR DESCRIPTION
**Pull request by Suyun114**
Topic Description
-----------------

sqlmap: new, 1.7.12

Package(s) Affected
-------------------

```
app-penetration/sqlmap
```

Security Update?
----------------

No

Test Build(s) Done
------------------

- [ ] Architecture-independent `noarch`

### CODE REVIEW

File: app-penetration/sqlmap/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,7 @@
+PKGNAME=sqlmap
+PKGSEC=web
+PKGDEP="python-3"
+PKGSUG="perl"
+PKGDES="Automatic SQL injection and database takeover tool"
+
+ABHOST=noarch
```
Comment (MingcongBai) [inline app-penetration/sqlmap/autobuild/defines]: Missing newline. Please check your editor configuration.

File: app-penetration/sqlmap/spec [hunk 2]
```diff
@@ -0,0 +1,4 @@
+VER=1.7.12
+SRCS="tbl::https://github.com/sqlmapproject/sqlmap/archive/$VER.tar.gz"
+CHKSUMS="sha256::74c20a05ac2e66c5302f69e854769e4fd23d4af460cb0e17863306355f18c70c"
+CHKUPDATE="anitya::id=14901"
```
Comment (MingcongBai) [inline app-penetration/sqlmap/spec]: Missing newline. Please check your editor configuration.

File: app-penetration/sqlmap/spec [hunk 3]
```diff
@@ -0,0 +1,4 @@
+VER=1.7.12
+SRCS="tbl::https://github.com/sqlmapproject/sqlmap/archive/$VER.tar.gz"
```
Comment (MingcongBai) [inline app-penetration/sqlmap/spec]: Please use Git source instead, as automatically generated tarballs may be re-generated - making them unreliable in the long run.

```
SRCS="git::commit=tags/$VER::https://github.com/sqlmapproject/sqlmap"
CHKSUMS="SKIP"
```

File: app-penetration/sqlmap/autobuild/defines [hunk 4]
```diff
@@ -0,0 +1,7 @@
+PKGNAME=sqlmap
+PKGSEC=web
```
Comment (MingcongBai) [inline app-penetration/sqlmap/autobuild/defines]: Web?

Comment (MingcongBai) [reply app-penetration/sqlmap/autobuild/defines]: This is likely better described in the `database` section.

File: app-penetration/sqlmap/autobuild/defines [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+PKGDEP="python-3"
+PKGSUG="perl"
```
Comment (MingcongBai) [inline app-penetration/sqlmap/autobuild/defines]: Why not DEP?

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Comment by jiegec** [comment]
1.8 is out.

File: app-penetration/sqlmap/autobuild/overrides/usr/bin/sqlmap [hunk 6]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+cd /opt/sqlmap
+python sqlmap.py "\$@"
```
Comment (jiegec) [inline app-penetration/sqlmap/autobuild/overrides/usr/bin/sqlmap:3]: What if we directly run /opt/sqlmap/sqlmap.py?

**Pull request closed without merging**